src/main/java/hu/icellmobilsoft/atr/sample/rest/LoadDataImpl.java

```diff
@@ -0,0 +1,47 @@
+package hu.icellmobilsoft.atr.sample.rest;
+
+import java.io.IOException;
+
+import javax.xml.stream.XMLStreamException;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.JsonParser;
+
+import hu.icellmobilsoft.atr.sample.util.XSDValidator;
+
+public class LoadDataImpl implements ILoadData {
+
+    @Override
+    public parseXml loadFromXml(String xmlFileName) {

```

Comment:
visszatérése legyen void itt és az interfaceben is, ami itt van az action logika, itt csak az action meghívásának kéne szerepelnie így :  

SamplePatientAction simplePatientAction = new SamplePatientAction();
    private static final String XML_FILE = ".....xml";
    private static final String JSON_FILE = "......json";

    @Override
    public void loadFromXml() throws BaseException {
        simplePatientAction.loadFromXml(XML_FILE);
    }

    @Override
    public void loadFromJson() throws BaseException {
        simplePatientAction.loadFromJson(JSON_FILE);
    }